### DISCUSSION TITLE
Open external link in new tab

### DISCUSSION THREAD

**Discussion by kristhayer11** [thegetty/quire#193] [Get Help] [answered]
Is there a way to make all EXTERNAL links open in a new tab, and all INTERNAL links open within the same window?
I checked Quire's documentation—it says that the default is that external links open in new pages, but that does not seem to be the case with my project. So, I tried this change in the config.yml: hrefTargetBlank: false
That did nothing. 

Is there any way to make a global change, internal vs. external links?

**Comment by geealbers** [comment, marked as answer]
Hi @kristhayer11, good question. This is something that changed with Hugo and we haven't yet integrated and documented for Quire.

It's explained in Hugo’s documentation at https://gohugo.io/getting-started/configuration-markup#markdown-render-hooks, but in short ....

In the layouts directory of your project, or in the layouts directory of your project's theme, you'll need to create a new directory called `_default` and then in that, another new directory called `_markup` and then in that, a new file called `render-link.html` with the following content:

```html
<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
```

This controls how links are rendered from Markdown and there's a bit of logic in there that says that if the link starts with `http` then it's an external link and should be opened in a new window.

**Reply by kristhayer11** [reply]
Interesting, I will experiment and see what happens. Thank you!

**Reply by mmpadilla15** [reply]
@kristhayer11 Just wondering if you were successful in this?
